src/main/java/no/entur/uttu/graphql/LinesGraphQLSchema.java

```diff
@@ -917,8 +912,7 @@ private void initCommonTypes() {
         .field(versionField)
         .field(newFieldDefinition().name(FIELD_NAME).type(multilingualStringObjectType))
         .field(newFieldDefinition().name("legalName").type(multilingualStringObjectType))
```

Comment:
legalName is not used in frontend

Reply:
[d98b3eb](https://github.com/entur/uttu/pull/432/commits/d98b3ebea2362905d4e09e0e4d2fc1331c9311e3)